bundles/org.openhab.binding.goecharger/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="goecharger"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="goe">
+		<label>Go-eCharger Binding Thing</label>
+		<description>Go-eCharger thing that represents the wallbox configuration and readings</description>
+
+		<channels>
+			<channel id="maxAmpere" typeId="amp" />
+			<channel id="pwmSignal" typeId="pwm" />
+			<channel id="error" typeId="err" />
+			<channel id="voltageL1" typeId="vl1" />
+			<channel id="voltageL2" typeId="vl2" />
+			<channel id="voltageL3" typeId="vl3" />
+			<channel id="currentL1" typeId="cl1" />
+			<channel id="currentL2" typeId="cl2" />
+			<channel id="currentL3" typeId="cl3" />
+			<channel id="powerL1" typeId="pl1" />
+			<channel id="powerL2" typeId="pl2" />
+			<channel id="powerL3" typeId="pl3" />
+			<channel id="phases" typeId="pha" />
+			<channel id="sessionChargeConsumptionLimit" typeId="scl" />
+			<channel id="sessionChargeConsumption" typeId="scs" />
+			<channel id="totalConsumption" typeId="eto" />
+			<channel id="allowCharging" typeId="alw" />
+			<channel id="cableEncoding" typeId="cbl" />
+			<channel id="temperature" typeId="tmp" />
+			<channel id="firmware" typeId="fmw" />
+			<channel id="accessState" typeId="ast" />
+		</channels>
+
+		<config-description>
+			<parameter name="ip" type="text" required="true">
+				<label>IP Adress</label>
+				<description>The IP adress of the Go-eCharger</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" required="false">
+				<label>Refresh Interval</label>
+				<description>Refresh interval for acquiring data from Go-eCharger in seconds</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="ast">
+		<item-type>String</item-type>
+		<label>Access State</label>
+		<description>Current access state of the go-eCharge</description>
+	</channel-type>
+	<channel-type id="amp">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Maximum Current</label>
+		<description>Max ampere per phase allowed to use for charging</description>
+		<state pattern="%d %unit%" readOnly="false"></state>
+	</channel-type>
+	<channel-type id="pwm">
+		<item-type>Number</item-type>
+		<label>PWM signal status</label>
+		<description>Signal status for PWM signal</description>
```

Comment:
The description allows for explaining what this abbreviation stands for so people uncommon to it don't have to look it up:

```suggestion
		<description>Pulse-width modulation signal status</description>
```